jota/src/test/java/org/iota/jota/IotaLocalPoWTest.java

```diff
@@ -56,17 +76,17 @@ public void shouldSendTransfer() throws ArgumentException {
     @Test
     public void shouldHaveEqualOrderTrytes() throws Exception {
         GetAttachToTangleResponse localResponse = iotaClient.attachToTangle(
-                TEST_SEED1, TEST_SEED1, 
+                TEST_SEED1, TEST_SEED1,
                 11, TEST_TRYTES_VALUE_TRANSFER);
-        
+
         iotaClient = new IotaAPI.Builder().config(new FileConfig()).build();
         GetAttachToTangleResponse remoteResponse = iotaClient.attachToTangle(
-                TEST_SEED1, TEST_SEED1, 
+                TEST_SEED1, TEST_SEED1,
                 11, TEST_TRYTES_VALUE_TRANSFER);
-    
+
         for (int i = 0; i < TEST_TRYTES_VALUE_TRANSFER.length; i++) {
-            assertEquals(new Transaction(localResponse.getTrytes()[i]).getValue(),
-                    new Transaction(remoteResponse.getTrytes()[i]).getValue());
+            assertEquals(new Transaction.Builder().buildWithTrytes(localResponse.getTrytes()[i]).getValue(),
```

Comment:
Codacy found an issue: [JUnit assertions should include a message](https://app.codacy.com/manual/Iota-java/iota-java/pullRequest?prid=6319461)